app/src/main/java/org/kinecosystem/kinit/KinitApplication.kt

```diff
@@ -52,6 +56,9 @@ class KinitApplication : Application(), DataStoreProvider {
         wallet.retrieveTransactions()
         wallet.retrieveCoupons()
         userRepository.isFreshInstall = false
+
+        val picasso = Picasso.Builder(this).memoryCache(LruCache(CACHE_SIZE)).build()
```

Comment:
do we need to set the memory cache size? 
Apparently picasso uses 1/7th of the available heap for it's LRU. I think staying with this default is safer than using a static number.

Reply:
Not a must, but since we're prefetching N images into memory it feels safer to be explicit about how much memory we want to use rather than rely on default values that might change.

Reply:
ok - I think it's safer to increase that a little to 7M 
But note that in most case we are actually requesting a lot less than what Picasso would ask